src/sentry/SentryTransactionHub.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/react-native'
+import { Transaction } from '@sentry/types'
+import { SentrySpan, SentrySpans } from 'src/sentry/SentrySpans'
+
+export const SentryTransactionHub = {
+  transactions: [] as Array<Transaction>,
+  startTransaction(name: SentrySpan) {
+    const transaction = Sentry.startTransaction({ ...SentrySpans[name], trimEnd: true })
+    this.transactions.push(transaction)
```

Comment:
i was a little surprised to see `this`, usually that's used with classes when there is an instance...can the `transactions` just be a variable in this file, where it's not exported? i think that achieves the same effect?